doc/bgm.md

```diff
@@ -0,0 +1,96 @@
+# Looping Background Music
```

Comment:
add this file to the doc/REAME.md list under Topics. That will add it to the flame docs site! You can use nesting if desired

Reply:
added!